src/app/features/reporter/reporter-dialog.component.html

```diff
@@ -42,29 +39,134 @@
 			</div>
 			<div class="col-md-9">
 				<test-group-combobox #testGroupComboBox [multipleSelection]="true"
-														 [(multipleSelectedItemList)]="selectedTestGroups"
-														 [isDisabled]="selectedTestCycleId!==undefined"></test-group-combobox>
+					[(multipleSelectedItemList)]="selectedTestGroups" [isDisabled]="selectedTestCycleId!==undefined">
+				</test-group-combobox>
 			</div>
 		</div>
 		<div class="row mt-2">
 			<div class="col-md-3">
 				<label for="newtestcycle" class="col-form-label">New Test Cycle name</label>
 			</div>
 			<div class="col-md-9">
-			<input type="text" class="form-control" id="newtestcycle" name="newtestcycle" [(ngModel)]="nameForNewTestCycle" [disabled]="selectedTestCycleId!==undefined">
+				<input type="text" class="form-control" id="newtestcycle" name="newtestcycle" [(ngModel)]="nameForNewTestCycle"
+					[disabled]="selectedTestCycleId!==undefined">
 			</div>
 		</div>
 		<div class="row mt-2">
 			<div class="col-md-3">
 				<label for="actualresults" class="col-form-label">Actual Results</label>
 			</div>
 			<div class="col-md-9">
-				<textarea id="actualresults" name="actualresults" class="slab-textarea w-100 slab-textarea-vertical-resize" rows="14" [(ngModel)]="actualResults"></textarea>
+				<textarea id="actualresults" name="actualresults" class="slab-textarea w-100 slab-textarea-vertical-resize"
+					rows="8" [(ngModel)]="actualResults"></textarea>
 			</div>
 		</div>
 	</form>
+	<div *ngIf="totalTestsRun > 0 && !areResultsReady()">Uploading...</div>
+	<div class="row mt-4 mb-4 results-container" *ngIf="areResultsReady()">
+		<div class="col-md-12 mt-4 mb-4">
+			<h4 class="mb-4">Upload Results</h4>
+			<table class="table table-bordered results-table">
+				<tbody>
+					<tr>
+						<td class="font-weight-bold">Total</td>
```

Comment:
Please, update the descriptions as follows 

- Total of Test Runs to upload 
- Number of Test Runs with Pass status 
- Number of Test Runs with Failed status 
- Number of Test Runs that exist in Jama but are not updated due to an error 
- Number of Test Runs that does not exist in Test Cycle in Jama 
- Number of Test Runs that are pending in the selected Test Cycle